Components/GridListSteps/_style.scss

```diff
@@ -15,29 +26,40 @@
   }
 
   .stepsList {
-    counter-reset: section;
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: center;
-    list-style: none;
-    margin: -20px -10px 0;
-    padding: 0;
-  }
+    @include grid(
+      $columns: 1,
+      $gutter-horizontal: $step-list-spacing,
+      $gutter-vertical: $step-list-spacing
+    );
+    @include reset-list;
 
-  .step {
-    max-width: 400px;
-    padding: 20px 10px 0;
+    align-items: center;
+    counter-reset: section;
+    flex-direction: column;
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
I assume this is about centering the grid items. If so, could we make use of the mixin `$alignment` option?

Reply:
`$alignment` option of the `grid` mixin deals only with `justify-content` property. 
Marked code, supplements the code generated by the mixin.